documentation/accessibility.md

```diff
@@ -93,6 +89,51 @@ When manually testing the application, it is good for you to ask yourself some q
   - Does the focus return to the element that triggered the opening of the dialog when closing?
 - Are you able to freely navigate the application without being "trapped"?
 
+### Testing Steps
+
+- Select and document a representative sample of pages (with URLs) and features for testing.
+- List pages to be tested with urls
+- Document Automated tools and AT configurations (Operating System, Browser, Technology)
+- Confirm the checks below with Keyboard, colour contrast checker and automated testing tools.
+- Review pages using the AT chosen configuration.
+
+#### Non-Technical checks
+
+- [ ] Confirm that the heading structure is hierarchical.
+- [ ] Check for consistent use of white space so pages are easy to scan.
+- [ ] Review non-text elements for information or functional value, proper labelling and/or alternative text for non-text elements with value.
+- [ ] Confirm that objects with informative or functional value have meaningful alternative or ARIA labels.
+- [ ] Review colour contrast for a contrast ratio of 4:5:1 between foreground and background. 
+- [ ] Review colour contrast for perceived colour contrast in addition to computed values. 
+- [ ] Confirm link texts are descriptive and that their "underline" signifier is present. Alternatively, they should have a contrast ratio of 3:1.
+- [ ] Confirm that key pages do not rely solely on colour to convey meaning or instruction. 
+
+#### Technical checks
+
+- [ ] Test for the presence of proper landmarks roles and labels (e.g., Main, Nav, aside, section , header, footer)
+- [ ] Confirm that all components/widgets have appropriate ARIA roles.
+- [ ] Review page titles and headings for descriptive titles and proper structure.
+- [ ] Review reading order and focus management (focus order, component triggers, modals, etc.). 
+- [ ] Confirm that meaningful graphical elements (those with informational value) have a contrast value of 3:1.
+- [ ] Confirm that touch targets are usable (24pi x24 pi vs 48pi x 48pi range)
+- [ ] Create a page example with text size enlarged to 200% to test text resizing and reflow. 
+- [ ] Test any new selectable components (toggle buttons, radio buttons, checkboxes, list boxes, text boxes, etc.) to see if it triggers an expected change in context.
+- [ ] Confirm if users have multiple ways to navigate to each page in the pattern. 
+- [ ] Check French-language pages for proper default language. 
+- [ ] Confirm that the platform does not require users to re-enter information they have already provided (e.g., in a previous step of the RoD process).
+
+#### Plain Language confirmations
+
+- [ ] Confirm that the copy reflects the light mode with fidelity.
```

Comment:
I'm not sure what this means.  Copy reflects light mode?

Reply:
We developed this list a bit ago and I don't remember what we meant so I just removed it 😅 

[fix formatting, improve clarity](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9135/commits/07a06df3e5f8dc7dfe78f67017c906b740d6822f)